Homework submission Ross O'Farrell

rossjimm opened the pull request:
A simple website. A complex man.

rossjimm committed:
Homework.

Very simple website for a very complex man!

bmwertman commented:
Nice job @rossjofarrell1983. Overall it looks good. I have one fix that I want you to make though.

🔨 Please review how to correctly indent your HTML and update both of your HTML pages.

Remember only elements that are nested inside other elements get tabbed in. Let me know if you have any questions.

rossjimm committed:
indentation adjustment

Hope this is better!

rossjimm commented:
Hey Brad I changed and saved my sublime text files. It looks like it hasn't
changed the github-desktop file though. Maybe we can look at it this
evening and see where I'm going wrong? Thanks!

Ross

On Mon, May 23, 2016 at 9:23 AM, Brad Wertman notifications@github.com
wrote:

> Nice job @rossjofarrell1983 https://github.com/rossjofarrell1983.
> Overall it looks good. I have one fix that I want you to make though.
> 
> 🔨 Please review how to correctly indent your HTML and update both of
> your HTML pages.
> 
> Remember only elements that are nested inside other elements get tabbed
> in. Let me know if you have any questions.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/FEWD87/homework-01-famous-person-resume-website/pull/10#issuecomment-220978739

bmwertman commented:
👍 I see your changes @rossjofarrell1983. Everything looks good.
